semperfiwebdesign/all-in-one-seo-pack
# Fix Divi icon set

Issue #2555

## Proposed changes

This fixes an issue where icons in Divi disappeared.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

1. Install the latest Divi theme (see Google Drive).
2. Go to Theme Options > Builder > Advanced and enable the Classic Editor.
3. Create a new Post or Page, choose Use the Divi Builder, then click Switch back to the Classic Editor in the bottom right of the Divi box.
4. Add a Blurb element, open the Image & Icon section and then enable Use Icons
5. Check and see if all icons appear again (no blank squares).
